Week5/SOLUTION/week_5.py

```diff
@@ -0,0 +1,32 @@
+from csv import *
+
+header = ['SID','SNAME','CITY','CONTACT']
+
+file_record=[
+    [1,'Meet','Surat',1234567890],
+    [2,'Om','Bardoli',1213141516],
+    [3,'Sai','Vyara',8989890000],
+    [4,'Ram','Surat',1234500001],
+    [5,'Harsh','Surat',7227848152]
+    ]
+
+main_list=[]
+for i in range(5):
+    sid=int(input("Enter Student Id :"))
+    sname=input("Enter Student Name :")
+    scity=input("Enter Student City :")
+    scontact=int(input("Enter Student Contact :"))
+    sub_lis=[sid,sname,scity,scontact]
+    main_list.append(sub_lis)
+
+file_record.extend(main_list)
+
+with open('student.csv','w',newline="") as file_create:
+    file_date = writer(file_create)
+    file_date.writerow(header)

```

Comment:
Why do we use the "writerow( )" or "writerows( )" method as "variable_name.writerow()" or 
"variable_name.writerows()"when working with the csv.writer() in python's csv module, instead of using it as "csv.writerow()"or "csv.writerows( )" ?

Reply:
because i import file in such a way that i don't need to write module name for accessing its class and function

Reply:
No ,Sorry for confusion you have from my question. but,
my question is that why we doesn't call writerow and writerrows function using csv module instead of why we use variable name which holdes writer( ) function statement?

for example in above code , you call writerow( ) like file_date .writerow(header)
but why  not like this csv.writerow(header)?

Reply:
Because we can use writerow or writerows only with file varaible

Reply:
But, why only with file variable ?

Reply:
Because File variable is like a pointer to the file in which we want to write the data